frontend/src/components/HostMetricsDetail/index.tsx

```diff
@@ -0,0 +1,170 @@
+									bordered
+									className="infra-monitoring-tags"
+								>
+									{host.active ? 'ACTIVE' : 'INACTIVE'}
+								</Tag>
+								<Tag className="infra-monitoring-tags" bordered>
+									{host.os}
+								</Tag>
+								<div className="progress-container">
+									<Progress
+										percent={Number((host.cpu * 100).toFixed(1))}
+										size="small"
+										strokeColor={((): string => {
+											const cpuPercent = Number((host.cpu * 100).toFixed(1));
+											if (cpuPercent >= 90) return Color.BG_SAKURA_500;
+											if (cpuPercent >= 60) return Color.BG_AMBER_500;
+											return Color.BG_FOREST_500;
+										})()}
+										className="progress-bar"
+									/>
+								</div>
+								<div className="progress-container">
+									<Progress
+										percent={Number((host.memory * 100).toFixed(1))}
+										size="small"
+										strokeColor={((): string => {
+											const memoryPercent = Number((host.memory * 100).toFixed(1));
+											if (memoryPercent >= 90) return Color.BG_CHERRY_500;
+											if (memoryPercent >= 60) return Color.BG_AMBER_500;
+											return Color.BG_FOREST_500;
+										})()}
+										className="progress-bar"
+									/>
+								</div>
+							</div>
+						</div>
+					</div>
+
+					<Radio.Group
+						className="views-tabs"
+						onChange={handleModeChange}
+						value={selectedView}
+					>
+						<Radio.Button
+							className={
+								// eslint-disable-next-line sonarjs/no-duplicate-string
+								selectedView === VIEW_TYPES.CONTAINERS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.CONTAINERS}
+						>
+							<div className="view-title">
+								<Package2 size={14} />
+								Containers
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.PROCESSES ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.PROCESSES}
+						>
+							<div className="view-title">
+								<ChevronsLeftRight size={14} />
+								Processes
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.METRICS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.METRICS}
+						>
+							<div className="view-title">
+								<BarChart2 size={14} />
+								Metrics
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.LOGS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.LOGS}
+						>
+							<div className="view-title">
+								<ScrollText size={14} />
+								Logs
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.TRACES ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.TRACES}
+						>
+							<div className="view-title">
+								<DraftingCompass size={14} />
+								Traces
+							</div>
+						</Radio.Button>
+					</Radio.Group>
```

Comment:
nit: can we define a config enum for these tabs and loop through it here to render the tabs?